src/app/pages/session-detail/session-detail.page.ts

```diff
@@ -169,7 +169,9 @@ export class SessionDetailPage implements OnInit {
     this.dismissWhenBack = true;
   }
   ionViewWillLeave(){
-    if(!this.skipWhenDelete){this.snackbarRef = this.toaster.dismiss()}
+    if(!this.skipWhenDelete){this.snackbarRef = this.toaster?.dismiss().catch(err =>{
```

Comment:
here also try something like above, remove consoles and all

Reply:
> here also try something like above, remove consoles and all
bro this toast is different. this is just a console error. to remove the console error we have to catch the error. so we did